Makefile

```diff
@@ -139,6 +139,9 @@ retest: testenv
 	@${MAKE} -C tests retest only=$(only) IDRIS2=${TARGET} IDRIS2_PREFIX=${TEST_PREFIX}
 
 
+test-installed: testenv
```

Comment:
What's the goal of this target? To my mind you want the compiler test suite to be entirely representative of any changes to the compiler living in the working directory. Is the goal to see how new test cases play out against an older version of the compiler?

Reply:
this target runs the tests with the **installed** version of Idris, i.e. this target detects the faulty installation.

let me know if it's not needed and i'll update the PR.

i have various changes planned for the build system, but i'm not sure this is the right place to discuss that, though. i will open PR's and/or write to the mailing list.

Reply:
I'm not totally convinced that testing an installed version of Idris is very useful, but if that's the goal then this target should probably work without relying on build artifacts (so testing is either totally dependent upon the local build of the Idris compiler or entirely dependent upon the installed Idris compiler but not a combination of the two).

Right now, `make clean && make test-installed` does not run because `make-installed` is not fully independent of the local build of the Idris compiler.

Reply:
oh, good point! i have fixed the PR to work that way.

but i'm still open for removing that target altogether, if it's not desirable. note though, that the normal tests didn't catch this issue with the installation.